.travis.yml

```diff
@@ -35,6 +40,7 @@ install:
     - gulp
     - make install-test
     - pip install coveralls
+    - if [[ $DJANGO_VERSION == 1.9 ]]; then pip install django==1.9; fi
```

Comment:
just install django in proper version (we could test django dev as well and don't want to if every case)